README.md

```diff
@@ -1 +1,102 @@
+## Using Git in VS Code
+In this guide I will tell you how you, yes you could use git through VS code making your life significantly easier than before. Am I saying you will never have to use console technically as far as VS code is concerned yes with pushing and pulling. You do not have to use console except when you would need to roll back after pushing changes, let’s get started.
 
+### Step 1
```

Comment:
Consider breaking your guide up into larger sections.

"Configuring your development branch"
"Committing your changes"
"Publishing to GitHub"

This is just an example of sections you could have. The point is that sections will help to better break-up the flow of steps.